llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -56,43 +52,64 @@ class Extension<
 
     // The FMV priority
     int FMVPriority = _FMVPriority;
+
+    // Indicates if the extension is available on the command line.
+    string IsFMVOnly = _IsFMVOnly;
```

Comment:
```suggestion
    bit IsFMVOnly = 0;
```

Reply:
We want the ExtensionInfo field to be of type bool. Can we do the type conversation in the emitter? If so okay